src/integration-test/javascript/e2e/homepageSpec.js

```diff
@@ -0,0 +1,7 @@
+describe('Demo Protractor Test', function () {
+    it('Should render the home.html partial', function () {
+        browser.get('/');
+
+        expect(element(by.tagName('p')).getText()).toBe('Rendered Courtesy of AngularJS!');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["expect" is not defined.](https://www.codacy.com/app/martzrp/booktrackr/file/2628357651/issues/source?bid=3283479&fileBranchId=3317266#l5)